charts/rancher-cis-benchmark/v0.0.1/values.yaml

```diff
@@ -22,9 +22,14 @@ debugWorker: false
 debugTime: "infinity"
 
 image:
-  repository: rancher/security-scan
-  tag: v0.1.9
-  pullPolicy: Always
+  securityScan:
+    repository: rancher/security-scan
```

Comment:
We need to change the tags in templates/pod.yaml as well for rancher/security-scan image

Reply:
Yeah, I changed the corresponding pod.yaml

Reply:
@StrongMonkey she means in here: https://github.com/rancher/system-charts/blob/dev/charts/rancher-cis-benchmark/v0.0.1/templates/pod.yaml#L68

as per her PR: https://github.com/rancher/system-charts/pull/183/files#diff-034d4bcb31f271b5aef68335ed19b67bL68

Reply:
:+1: missed that